test/fixtures/output/flink/shell/reset-all-keys.golden

```diff
@@ -26,6 +26,7 @@ configuration has been reset successfully.
 +------------------------+---------------------+
 |          Key           |        Value        |
 +------------------------+---------------------+
+| client.output-format   | standard (default)  |
```

Comment:
Add `test/fixures/output/flink/shell` to CODEOWNERS so you don't need to ask for a review?